lib/diagnostic.rb

```diff
@@ -74,7 +98,17 @@ def example_diff
   end
 
   describe 'PATCH update' do
-    # your test(s) here
+    before(:each) do
+      patch :update, id: Example.first.id, example: example_diff, format: :json
+    end
+
+    it 'is successful' do
+      expect(response).to be_success
+    end
+
+    it 'renders a JSON response' do
+      expect_json
```

Comment:
Need to `JSON.parse` the response from the `PATCH` request with:

`expect(JSON.parse(response.body)).not_to be_nil`